File: src/java/beans/Place.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package beans;

/**
 *
 * @author OTTO
 */
public class Place {

    private int idPlace;
    private int numero;
    private Boolean etat;
    private int idSection;
    static int cmp = 0;

    public Place(int idPlace, int numero, Boolean etat, int idSection) {
        this.idPlace= idPlace;
        this.numero = numero;
        this.etat = etat;
        this.idSection = idSection;
    }

    public Place(int numero, Boolean etat, int idSection) {
        this.idPlace = cmp++;
        this.numero = numero;
        this.etat = etat;
        this.idSection = idSection;
    }

    public int getId() {
        return idPlace;
    }

    public void setId(int idPlace) {
        this.idPlace = idPlace;
    }

    public int getNumero() {
        return numero;
    }

    public void setNumero(int numero) {
        this.numero = numero;
    }

    public Boolean getEtat() {
        return etat;
    }

    public void setEtat(Boolean etat) {
        this.etat = etat;
    }

    public int getIdSection() {
        return idSection;
    }

    public void setIdSection(int idSection) {
        this.idSection = idSection;
    }

    @Override
    public String toString() {
        return "Place{" + "id=" + idPlace + ", numero=" + numero + ", etat=" + etat + ", idSection=" + idSection + '}';
    }
    
    

}

File: web/script/Place.js
/* 
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
$(document).ready(function () {
    $.ajax({
        url: "PlaceController",
        data: {op: "load"},
        type: 'POST',
        success: function (data, textStatus, jqXHR) {
            
            remplir(data);
        },
        error: function (jqXHR, textStatus, errorThrown) {
            alert(textStatus);
        }
    });
    $("#content").on("click", "#update", function ()
    {
        id = $(this).closest("tr").find("td").eq(0).text();
        etat = $(this).closest("tr").find("td").eq(2).text();
        
        $.ajax({
            url: "PlaceController",
            data: {op: "updateEtat", id: id , etat : etat},
            
            type: 'POST',
            success: function (data, textStatus, jqXHR) {
                remplir(data);
            },
            error: function (jqXHR, textStatus, errorThrown) {
                console.log(textStatus);
            }
        });

    });

    
    function remplir(data) {
        var ligne = "";
        data.forEach(function (e) {
            
            ligne += "<tr><td>" + e.idPlace + "</td><td>" + e.numero + "</td><td>" + e.etat + "</td><td>"+e.idSection + "</td><td><button class='btn btn-warning' id='update'>free/full</button></td></tr>";
           
        });
        $("#content").html(ligne);
        
    }
});